source/features/center-reactions-popup.css

```diff
@@ -2,7 +2,12 @@
 	top: 100% !important;
 	left: 50% !important;
 	right: auto !important;
```

Comment:
It looks like we only need `bottom: auto` here so we make the position consistent. There's no particular reason to keep the popup above or below, so we can just stick to "below"

Reply:
Also none of these `important`s seem to be necessary except the transform.

Also if you just replace the transform with `translate: -50%` you preserve the popping animation too.

Reply:
I think the bottom emoji selector dropdown should open to the top (like GitHub's native one). When you scroll just so that the emoji selector is the last visible thing on the page, you'd otherwise have to scroll again to click the dropdown.

Reply:
> Also none of these importants seem to be necessary except the transform

True, except for `display: grid` on Firefox

> I think the bottom emoji selector dropdown should open to the top (like GitHub's native one)

+0

Reply:
@FloEdelmann GitHub’s positioning does indeed make some sense, they try to keep the dropdown inside the comment area, which is necessary to avoid this on tight viewports (this PR, right now)

<img width="193" alt="Screen Shot 7" src="https://user-images.githubusercontent.com/1402241/189525593-1dff4d91-5ec9-4379-a622-c9ff5c559778.png">


I would not mind going back to the positioning and try this instead, covering the toggle altogether and keeping it aligned:

<img width="204" alt="Screen Shot 8" src="https://user-images.githubusercontent.com/1402241/189525686-2fbeae51-e809-49de-b63b-db3ab9036251.png">

It does require further CSS

Reply:
@cheap-glitch perhaps you can get away with using the standard `top: 50, left: 50, translate: -50, -50` technique and then adjusting it where necessary